### ISSUE TITLE
Destructing import broken on patch v3.0.1

### ISSUE THREAD

**Issue by navarroaxel** [mongodb/js-bson#263]
In `bson@3.0.0` this code works perfectly:

```js
import {ObjectID} from 'bson';

const generateId = () => ObjectID().toString();

export default generateId;
```

In version `bson@3.0.1` the module does not export the BSON modules correctly, I have to made this to get the `ObjectID`:
```js
import * as bson from 'bson';
const {ObjectID} = bson.default;
```

I think the issue is in `bson@3.0.1` because `webpack` is using the file `dist/bson.esm.js` and the export is not correct:
```js
var ObjectID_1$1 = objectid;
bson.ObjectID = ObjectID_1$1
var jsBson = bson;
export default jsBson;
```
I think the correct should be something like:
```js
export {ObjectID};
```
because the file is an `EcmaScript module`.

Versions:
```
node@8.11.3
npm@6.1.0
webpack@4.15.1
```

Tell me how can I help.

**Comment by navarroaxel** [comment]
A demo repository to test this bug: https://github.com/navarroaxel/webpack-bson-bug

**Comment by daprahamian** [comment]
@navarroaxel we just published v3.0.2, which should fix your issue.

**Comment by navarroaxel** [comment]
Thanks for the fast fix.
